frontend/src/dto/CallLogs/CallLogDTOs.ts

```diff
@@ -0,0 +1,23 @@
+export interface CallUpdateDTO {
+  receiverName?: string;
+  date?: string;
+  phoneNumber?: string;
+  description?: string;
+  receiverEmail?: string;
+  action?: string;
+  followUp?: boolean;
+  neverCallBack?: boolean;
+  callerEmail?: string;
+}
+
+export interface CallCreationDTO {
+  receiverName?: string;
+  date: string;
+  phoneNumber?: string;
+  description: string;
+  receiverEmail: string;
+  action: string;
```

Comment:
@nasaku898 Same comment as previous PR (action should be of type ENUM). Should we make a ticket to handle this?

Reply:
Yeah I think we can create a seperate ticket just so we can merge this one.